src/main/java/org/broadinstitute/hellbender/utils/help/HelpConstants.java

```diff
@@ -19,62 +19,99 @@ public static String forumPost(String post) {
+
+    public final static String DOC_CAT_READS = "Read Data Manipulation";  // Picard placeholder
+    public final static String DOC_CAT_READS_SUMMARY = "Tools that manipulate read data in SAM, BAM or CRAM format";
+
+    public final static String DOC_CAT_VARIANT = "VCF Tools";   // Picard placeholder
+    public final static String DOC_CAT_VARIANT_SUMMARY = "Tools for manipulating variants and associated metadata";
+
+    public final static String DOC_CAT_VARIANT_FILTERING = "Variant Filtering"; // Picard placeholder
+    public final static String DOC_CAT_VARIANT_FILTERING_SUMMARY = "Tools that filter variants by annotating the FILTER column";
 
-    public final static String DOC_CAT_FASTA = "Fasta File Tools";
-    public final static String DOC_CAT_FASTA_SUMMARY = "Tools for analysis and manipulation of files in fasta format";
+    public final static String DOC_CAT_VARIANT_EVALUATION = "Variant Evaluation and Refinement";   // Picard placeholder
+    public final static String DOC_CAT_VARIANT_EVALUATION_SUMMARY = "Tools that evaluate and refine variant calls, e.g. with annotations not offered by the engine";
 
-    public final static String DOC_CAT_INTERVALS = "Interval Tools";
-    public final static String DOC_CAT_INTERVALS_SUMMARY = "Tools for processing intervals and associated overlapping records";
+    public final static String DOC_CAT_VCF_MANIPULATION = "VCF Manipulation";   // Picard placeholder
+    public final static String DOC_CAT_VCF_MANIPULATION_SUMMARY = "Tools that manipulate variant call format (VCF) data";
 
-    public final static String DOC_CAT_READS = "SAM/BAM/CRAM Tools";
-    public final static String DOC_CAT_READS_SUMMARY = "Tools for manipulating read-level data (SAM/BAM/CRAM)";
+    //  End temporary Picard placeholders.
+
+    public final static String DOC_CAT_ANNOTATORS = "Variant Annotations";
+    public final static String DOC_CAT_ANNOTATORS_SUMMARY = "Available to HaplotypeCaller, Mutect2, VariantAnnotator and GenotypeGVCFs. " +
+            "See https://software.broadinstitute.org/gatk/documentation/article?id=10836";
+
+    public final static String DOC_CAT_BWA_MEM_UTILS = "Bwa Mem JNI binding Tools";
+    public final static String DOC_CAT_BWA_MEM_UTILS_SUMMARY = "Bwa Mem JNI binding Tools";
+
+    public final static String DOC_CAT_COVERAGE_ANALYSIS = "Coverage Analysis";
+    public final static String DOC_CAT_COVERAGE_ANALYSIS_SUMMARY = "Tools that count coverage, e.g. depth per allele";
+
+    public final static String DOC_CAT_CNV = "Copy Number Variant Discovery";
+    public final static String DOC_CAT_CNV_SUMMARY = "Tools that analyze read coverage to detect copy number variants.";
+
+    public final static String DOC_CAT_EXAMPLE = "Example Tools";
+    public final static String DOC_CAT_EXAMPLE_SUMMARY = "Example tools that show developers how to implement new tools";
+
+    public final static String DOC_CAT_METAGENOMICS = "MetaGenomics";
```

Comment:
seems like this would be clearer as PathogenDetection instead of MetaGenomics, but I could be wrong about that.

Reply:
"pathogen" is very judgemental -- there are a lot of applications of PathSeq that are not necessarily disease-related

Reply:
I wouldn't want to offend any beneficial microbes!

Reply:
The workflow tool names start with _Pathseq_ so folks will immediately know what Metagenomics refers to. @mwalker174 requested we use _metagenomics_ instead of _pathogen_.